From 836963d39e0699cbd73d3f1a19948207f69a05a5 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Wed, 15 Jun 2016 15:15:43 -0400
Subject: [PATCH 3/5] Do not disable vusb3v1 regulator

---
 drivers/phy/phy-twl4030-usb.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/drivers/phy/phy-twl4030-usb.c b/drivers/phy/phy-twl4030-usb.c
index 6ab230f..aa3908c 100644
--- a/drivers/phy/phy-twl4030-usb.c
+++ b/drivers/phy/phy-twl4030-usb.c
@@ -144,6 +144,9 @@
 #define PMBR1				0x0D
 #define GPIO_USB_4PIN_ULPI_2430C	(3 << 0)
 
+#define TWL4030_USB_SEL_MADC_MCPC	(1 << 3)
+#define TWL4030_USB_CARKIT_ANA_CTRL	0xBB
+
 /*
  * If VBUS is valid or ID is ground, then we know a
  * cable is present and we need to be runtime-enabled
@@ -402,7 +405,6 @@ static int twl4030_usb_runtime_suspend(struct device *dev)
 	__twl4030_phy_power(twl, 0);
 	regulator_disable(twl->usb1v5);
 	regulator_disable(twl->usb1v8);
-	regulator_disable(twl->usb3v1);
 
 	return 0;
 }
@@ -469,6 +471,10 @@ static int twl4030_phy_power_on(struct phy *phy)
 		twl4030_i2c_access(twl, 0);
 	schedule_delayed_work(&twl->id_workaround_work, 0);
 
+	twl4030_usb_write(twl, TWL4030_USB_CARKIT_ANA_CTRL,
+		twl4030_usb_read(twl, TWL4030_USB_CARKIT_ANA_CTRL) |
+			TWL4030_USB_SEL_MADC_MCPC);
+
 	return 0;
 }
 
-- 
2.7.4

